File: icpc/2017_RPC10/G.cpp
#include <bits/stdc++.h>
using namespace std;

int main () {
    ios_base::sync_with_stdio(0), cin.tie(nullptr);
    int T; cin >> T;
    while (T--) {
        string S;
        cin.ignore();
        getline(cin, S);

        istringstream iss(S);
        vector<int> a;
        for (int x; iss>>x; )
            a.push_back(x);

        int top = *max_element(a.begin(), a.end());
        int res = 0;
        bool start = 0;
        cout << '\n' << a.size() << endl;
        for (int x : a) {
            if (x == top) ++res, start = !start;
            else if (start) ++res;
            cout << x << ' ' << start << ' ' << res << endl;
        }
        cout << res << endl;
    }
}

File: icpc/2017_RPC10/G.py
T = int(input())
for _ in range(T):
    nums = list(map(int, input().split(' ')))
    res = 0
    start = 0
    top = max(nums)
    for x in nums:
        if x == top:
            res += 1
            start = not start
        elif start:
            res += 1
    print(res)

